Cargo.toml

```diff
@@ -21,3 +21,5 @@ codegen-units = 1
 [patch.crates-io]
 zcash_encoding = { path = "components/zcash_encoding" }
 zcash_note_encryption = { path = "components/zcash_note_encryption" }
+schemer = { git = "https://github.com/nuttycom/schemer.git", rev = "b3f7c8a3abb69b67d68258ae5284c5743bdb380b" }
+schemer-rusqlite = { git = "https://github.com/nuttycom/schemer.git", rev = "b3f7c8a3abb69b67d68258ae5284c5743bdb380b" }
```

Comment:
Why do these need to be forked? I cannot see any differences between these forks, and [schemer](https://crates.io/crates/schemer) / [schemer-rusqlite](https://crates.io/crates/schemer-rusqlite) 0.1.1, apart from https://github.com/aschampion/schemer/commit/cd224eb325f5bcfd9150520546a48f24ddf6e66b (which was made after the schemer 0.1.2 release but does not change behaviour at all).

Reply:
The changes are those represented by these PRs: https://github.com/aschampion/schemer/pull/13, https://github.com/aschampion/schemer/pull/14

Reply:
👍